libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -700,10 +836,6 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           },
           lifecycle: pruneAfterDays(970),
           progress: 0.25,
-          onExit: defineTemplateApi({
-            action: ApiModuleActions.validateApplication,
-            throwOnError: true,
-          }),
           roles: [
```

Comment:
Why taking out validateApplication here? We need it before applicant could send updated period to employers or VMST

Reply:
This was a mistake. I should not have been taking out. Thanks for noticing. 👍